pkg/datarepair/irreparabledb/irreparabledb.go

```diff
@@ -7,18 +7,24 @@ import (
 	"context"
 
 	"github.com/zeebo/errs"
-
-	"storj.io/storj/internal/migrate"
-	dbx "storj.io/storj/pkg/datarepair/irreparabledb/dbx"
-	"storj.io/storj/pkg/utils"
 )
 
 // Error is the default irreparabledb errs class
 var Error = errs.Class("irreparabledb error")
 
-// Database implements the irreparable RPC service
+// DB interface for database operations
+type DB interface {
```

Comment:
Move this into `datarepair.IrreparableDB`

Reply:
Done

Reply:
This move wasn't done, although... Let's leave as is... I suspect it would create cyclic dependencies.